modelina-website/src/components/playground/options/GeneralOptions.tsx

```diff
@@ -64,15 +66,25 @@ class GeneralOptions extends React.Component<
     }
   }
 
+  
+
+
   render() {
     return (
       <ul className="flex flex-col">
         <h3 className="text-lg font-medium leading-6 text-gray-900">
           General options
         </h3>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
-            <span className="mt-1 max-w-2xl text-sm text-gray-500">
+        <li className=' flex items-center mt-3'>
+          <InfoModal text="Output type :">
+            <p>
+              Additional information about the output type{' '}
+              <a href="https://example.com" target="_blank" rel="noopener noreferrer" className="text-blue-400 underline">Click here</a> to visit the website.
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium ipsam, ex quis veritatis totam quo. Cupiditate sequi laudantium corporis commodi.
```

Comment:
Alright, lets add some real info here then, and lets merge this 😄

I think we can do the following:

```
Change which kind of output you want to generate, remember it can happen that not all outputs are supported in the playground. For an updated list, head over to <a href="https://github.com/asyncapi/modelina#features">the main readme file</a>.
```

Feel free to change it as you see fit 🙂

Reply:
done !!